src/Components/LandingPage/NewAlert.tsx

```diff
@@ -1,16 +1,43 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-import { Alert, Text } from '@patternfly/react-core';
+import { Alert, AlertActionCloseButton, Text } from '@patternfly/react-core';
 
 export const NewAlert = () => {
-  return (
-    <Alert title="New in Images: Blueprints!">
-      <Text>
-        Blueprints make it easier for you to manage your images. Images expire
-        after two weeks, but blueprints last forever. Create a blueprint for
-        your “golden image”, modify it over time as your needs change, and use
-        it to build and deploy images on demand.
-      </Text>
-    </Alert>
-  );
+  const [displayAlert, setDisplayAlert] = useState(true);
+
+  useEffect(() => {
```

Comment:
Could we get rid of this useEffect if we just declared `alertDismissed` before the useState hook for displayAlert and initialized it like `const [displayAlert, setDisplayAlert] = useState(alertDismissed || true);`?

Reply:
This seems to be working on my machine, I was able to get rid of both useEffect hooks:
![image](https://github.com/osbuild/image-builder-frontend/assets/95542540/14ffb3c9-979f-4d17-aa8a-5af788424042)

Reply:
You're right, this works perfectly fine. I've pushed the changes.

Reply:
Wow, I'm glad we have this figured out! Now we can do all sorts of fun stuff with persistence across sessions!